### DISCUSSION TITLE
How to configure installed lsps?

### DISCUSSION THREAD

**Discussion by siduck** [williamboman/nvim-lsp-installer#664] [Q&A] [answered]
hi @williamboman , its been a while since I used lsp-installer. Im considering adding it in nvchad, but confused regarding this matter! Before there were docs regarding configuring installed lsp servers but now there arent any! 

From this I can assume that we can configure installed lsp-servers by lspconfig itself?

![image](https://user-images.githubusercontent.com/59060246/166604660-891c0f15-568d-4931-9b11-7ee2a0688205.png)

**Comment by bryant-the-coder** [comment, marked as answer]
Just install the server you want. Ignore lsp-installer and configure it like how you did previously

**Reply by siduck** [reply]
yea I found in the readme that lspconfig looks for a path specified by lsp-installer :thinking:

**Reply by bryant-the-coder** [reply]
Yup. As for windows its under `C:/Users/<username>/AppData/Local/nvim-data`

**Reply by williamboman** [reply]
Precisely, nvim-lsp-installer is no longer explicitly involved in setting up of servers (it is, but it happens behind the scenes), see https://github.com/williamboman/nvim-lsp-installer/discussions/636
